tracing/zipkin/http_test.go

```diff
@@ -0,0 +1,160 @@
+package zipkin_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"reflect"
+	"testing"
+
+	zipkinkit "github.com/go-kit/kit/tracing/zipkin"
+	kithttp "github.com/go-kit/kit/transport/http"
+	zipkin "github.com/openzipkin/zipkin-go"
+	"github.com/openzipkin/zipkin-go/model"
+	"github.com/openzipkin/zipkin-go/reporter/recorder"
+)
+
+const (
+	testName     = "test"
+	testBody     = "test_body"
+	testTagKey   = "test_key"
+	testTagValue = "test_value"
+)
+
+func TestHttpClientTracePropagatesParentSpan(t *testing.T) {
+	rec := recorder.NewReporter()
+	defer rec.Close()
+
+	tr, _ := zipkin.NewTracer(rec)
+
+	rURL, _ := url.Parse("http://test.com")
+
+	c := kithttp.NewClient(
```

Comment:
it's typical for Go kit to create a client and call Endpoint() on it.

Like this:
```
ep := kithttp.NewClient(...).Endpoint()
```